packages/forma-36-react-components/src/components/Typography/DisplayText/DisplayText.tsx

```diff
@@ -11,6 +11,7 @@ export interface DisplayTextProps {
   children?: React.ReactNode;
   style?: React.CSSProperties;
   testId?: string;
+  title?: string;
```

Comment:
I think we should just do `interface DisplayTextProps extends HTMLProps<HTMLElement> { .. }`, that way we won't need to redefine html props

Reply:
Yeah, it sounds better, thanks for the suggestion

Reply:
`HTMLProps` pollutes the props type with unrelated props like `href`, `as`, so I used `HTMLAttributes`